CMakeLists.txt

```diff
@@ -63,3 +66,7 @@ endif()
 pkg_config_append_libs(mc_rbdyn_urdf)
 
 setup_project_finalize()
+SETUP_PROJECT_CPACK()
```

Comment:
As noted in the documentation of the macro, this is kind-of outdated and better solution are built-in for archive generations.

Reply:
Also this line has been removed.